app/controllers/api/v1/products_controller.rb

```diff
@@ -0,0 +1,5 @@
+class Api::V1::ProductsController < Api::V1::ApplicationController
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/blyscuit/rails-example-one/blob/1d081ecd36c48dd47f0123e7164aa066b7a4d685/app/controllers/api/v1/products_controller.rb#L1"></span>Use nested module/class definitions instead of compact style.</td>
    </tr>
  </tbody>
</table>